H2MLauncher.UI/MainWindow.xaml

```diff
@@ -114,104 +117,259 @@
+                            <DataGridTextColumn Header="Players" 
+                                                SortMemberPath="ClientNum"
+                                                Binding="{Binding Occupation}"
+                                                CanUserResize="False">
+                                <DataGridTextColumn.ElementStyle>
+                                    <Style TargetType="TextBlock">
+                                        <Setter Property="HorizontalAlignment" Value="Right" />
+                                    </Style>
+                                </DataGridTextColumn.ElementStyle>
+                            </DataGridTextColumn>
+
+                            <!-- Updated Favorite Column with Clickable Feature -->
+                            <DataGridTemplateColumn Header="Favorite" CanUserResize="False">
+                                <DataGridTemplateColumn.CellTemplate>
+                                    <DataTemplate>
+                                        <Button Content="{Binding IsFavorite, Converter={StaticResource BooleanToStarConverter}}"
+                                                Command="{Binding DataContext.ToggleFavoriteCommand, RelativeSource={RelativeSource AncestorType=DataGrid}}"
+                                                CommandParameter="{Binding}"
+                                                HorizontalAlignment="Center" 
+                                                VerticalAlignment="Center"
+                                                FontSize="16"
+                                                Background="Transparent"
+                                                BorderThickness="0"
+                                                Cursor="Hand"/>
+                                    </DataTemplate>
+                                </DataGridTemplateColumn.CellTemplate>
+                            </DataGridTemplateColumn>
+
+
+
+
+
+                            <DataGridTextColumn Header="Ping" Binding="{Binding Ping}" CanUserResize="False">
+                                <DataGridTextColumn.ElementStyle>
+                                    <Style TargetType="TextBlock">
+                                        <Setter Property="HorizontalAlignment" Value="Right" />
+                                        <Setter Property="FontFamily" Value="Consolas" />
+                                        <Setter Property="FlowDirection" Value="RightToLeft" />
+                                        <Setter Property="Foreground" Value="{Binding Ping, Converter={StaticResource PingColorConverter}}" />
+                                    </Style>
+                                </DataGridTextColumn.ElementStyle>
+                            </DataGridTextColumn>
+                        </DataGrid.Columns>
+                    </DataGrid>
+
+
+                </TabItem>
+                <TabItem Header="Favorites">
```

Comment:
I was thinking, we could have the user save filters as tabs next to the favorite one (binding the collection of tabs). Then we will need some `ServerBrowserTabViewModel` to handle each to some limit. What do you think? also tagging @tobibodamer

Reply:
You mean that the user could save a certain filter i.e ([dom,war,dm],maxPing:70,minPlayers:5) and select them in the tab just to not enter the same filter over and over again ? Seems cool , and also, for a future these would be usefull for the matchmaking servers I think

Reply:
Let's make a seperate issue for that. I am currently implementing the server filter options, lets see first how that turns out

Reply:
Deferred task to issue #48